book/src/keymap.md

```diff
@@ -306,6 +306,15 @@ Keys to use within picker. Remapping currently not supported.
 | `Ctrl-v`                     | Open vertically   |
 | `Escape`, `Ctrl-c`           | Close picker      |
 
+## File Picker extra keymap
```

Comment:
Is there any reason these can't just go into the Picker section's table?

Reply:
The shortcut only works on FilePicker.

Reply:
Are you telling me that these shortcuts in the File Picker section work outside of the file picker?

https://github.com/helix-editor/helix/blob/ddbf03613d5a3dd64f3f5ba13795e70c29297837/book/src/keymap.md#L297-L309

Reply:
Yes, those shortcuts works on any picker and this is specific to file picker.

Reply:
Ohhhh, I see, sorry for my confusion. In that case, could we update the title to just "File picker", and update the wording in the sentence just below here to say that these are specific to the file picker?